### PR TITLE
Fix incorrect indicator area error message #494

### PR DESCRIPTION
**Pull request by temanbrcom**

### CODE REVIEW

**Comment by temanbrcom** [comment]
@grianbrcom 
@asatklichov

File: server/src/main/java/com/ca/lsp/core/cobol/preprocessor/sub/line/reader/impl/CobolLineReaderImpl.java [hunk 1]
```diff
@@ -46,14 +44,31 @@
...
-      compile("^(.{6})" + INDICATOR_FIELD + "(.{0,4})(.{0,61})(.{0,8})");
+  private static final int MAX_LINE_LENGTH = 80;
+  private static final String LONG_LINE_MSG = "Source text can not go past column 80.";
+  private static final String INCORRECT_LINE_FORMAT_MSG = "Unexpected indicator area content";
+  private static final Map<String, CobolLineTypeEnum> INDICATORS =
+      Map.of(
+          "*",
+          COMMENT,
+          "/",
+          COMMENT,
```
Comment (grianbrcom) [inline server/src/main/java/com/ca/lsp/core/cobol/preprocessor/sub/line/reader/impl/CobolLineReaderImpl.java:55]: Lines like:
```
"*", COMMENT,
"/", COMMENT,
```
will be easy to read.

**Review by grianbrcom — APPROVED** [review]
:shell:

File: server/src/main/java/com/ca/lsp/core/cobol/preprocessor/sub/line/reader/impl/CobolLineReaderImpl.java (hunk 1 above)
Comment (temanbrcom) [reply server/src/main/java/com/ca/lsp/core/cobol/preprocessor/sub/line/reader/impl/CobolLineReaderImpl.java:55]: It's true, but this is just an automated alignment

**Review by asatklichov — APPROVED** [review]
🏇

**Comment by zacanbrcom** [comment]
I started the review 🍭

**Review by zacanbrcom — APPROVED** [review]

**Review by zimlu02 — APPROVED** [review]

**Pull request merged**